cmd/process-agent/app/check.go

```diff
@@ -153,7 +153,12 @@ func runCheck(cfg *config.AgentConfig, ch checks.Check) error {
 		return fmt.Errorf("collection error: %s", err)
 	}
 
-	time.Sleep(1 * time.Second)
+	if ch.Name() == checks.ProcessEvents.Name() {
+		log.Info("Listening for process_events during 10s before running the check")
+		time.Sleep(10 * time.Second)
```

Comment:
I thought about it but I didn't want to add an arg that is only used by the `process_events` check. 
One solution is to get rid of this if/else block and add a duration argument that is `1s` by default and that we can override in case we're running the `process_events` check. What do you think about it?